Rakefile

```diff
@@ -4,9 +4,19 @@ require 'bundler'
 require 'bundler/setup'
 require 'rspec/core/rake_task'
 
+EXAMPLE_APP_BUNDLE_PATH = File.expand_path('../vendor/example_app-bundle', __FILE__)
+EXAMPLE_APP_HOME = 'spec/resources/example_app'
```

Comment:
As a matter of fact, yes. Or maybe no, but there's a reason why it's necessary for the former. And that is that the former path should be either relative to the latter or absolute, which might make it confusing to readers. The latter is relative to the current directory, which isn't as confusing.